modules/ROOT/pages/apikit-4-forgraphql.adoc

```diff
@@ -0,0 +1,19 @@
+= APIkit for GraphQL Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit"]
+
+GraphQL is a data query and manipulation language that enables API consumers to ask for the data that is only relevant to them. The APIkit for GraphQL module works on Anypoint Code Builder and it enables you to develop Mule applications. This module supports the implementation of GraphQL services, which use connectors to map GraphQL APIs to the existing backends where source data is stored.
+
+The APIkit scaffolder enables you to quickly implement projects by generating the initial skeleton of the application at design time based on the GraphQL schema. The application design maps queries, mutations, and types in your API to the Mule flows so you only specify how your data sources fulfill a particular data type.
+
+At run time, the APIkit for GraphQL router analyzes the query graph and orchestrates the execution of Mule flows based on the requests. The result of each flow execution is combined to return a result that mirrors the shape of the requested query.
+
+== See Also
+
+* xref:anypoint-code-builder::implement-a-graphql-api.adoc[Implement a GraphQL API specification]
+* xref:apikit-graphql-api-mapping.adoc[Mapping a GraphQL API to your Data Sources]
```

Comment:
```suggestion
* xref:apikit-graphql-api-mapping.adoc[Mapping a GraphQL API to Your Data Sources]
```

Reply:
If you're unsure about title capitalization, see the https://capitalizemytitle.com/ tool.

Reply:
Great! Thanks!